react/client/src/App.tsx

```diff
@@ -42,12 +44,31 @@ const theme = createMuiTheme({
 
 const App: React.FC = () => {
   const history = useHistory();
+  const { i18n } = useTranslation();
+
+  const handleClick = (language: any) => {
+    i18n.changeLanguage(language);
+  };
 
   return (
     <ThemeProvider theme={theme}>
       <Router basename={process.env.PUBLIC_URL}>
         <RoutingContextProvider>
           <AffirmationContextProvider>
+            <nav
+              style={{
+                width: '100%',
+                padding: '2rem 0',
+                backgroundColor: 'gray',
+                display: 'flex',
+                justifyContent: 'center',
+              }}
+            >
+              <button onClick={() => handleClick('en')}>ENG</button>
+              <button onClick={() => handleClick('esp')}>ESP</button>
+              <button onClick={() => handleClick('ko')}>KO</button>
+            </nav>
```

Comment:
A little language dropdown, neat!

Reply:
Yes, i'll comment it out for now until design team figures out how they want to implement it, if they want to implement it